packages/origin-ui-core/src/components/orders/MyOrders.tsx

```diff
@@ -2,20 +2,29 @@ import React from 'react';
 import { useSelector } from 'react-redux';
 import { OrderSide, OrderStatus } from '@energyweb/exchange-core';
 
-import { Order } from '../../utils/exchange';
-import { getOrders } from '../../features/orders/selectors';
+import { Order, Demand, DemandStatus } from '../../utils/exchange';
+import { getOrders, getDemands } from '../../features/orders/selectors';
 import { BidsTable } from './BidsTable';
 import { AsksTable } from './AsksTable';
 import { Box } from '@material-ui/core';
+import { DemandsTable } from './DemandsTable';
 
 export const MyOrders = () => {
+    const demands: Demand[] = useSelector(getDemands).filter(
+        (d) => d.status !== DemandStatus.ARCHIVED
+    );
     const orders: Order[] = useSelector(getOrders).filter(
         (o) => o.status === OrderStatus.Active || o.status === OrderStatus.PartiallyFilled
     );
     const asks = orders.filter((o) => o.side === OrderSide.Ask);
-    const bids = orders.filter((o) => o.side === OrderSide.Bid);
+    const bids = orders.filter(
+        (o) => o.side === OrderSide.Bid && !{}.hasOwnProperty.call(o, 'demandId')
```

Comment:
Possibly better solution would be to:

`o.demandId === undefined`

or

Maybe even better to create custom Orders object like

```
class Orders extends Array<Order> {
  constructor(orders: Order[]) {
    super(...orders);
  }

  public get bids() {
    return this.filter(o=>{logic here});
  }

  ...
}
```

and keep those filtering there, which improves the reusability etc.